zoom-recording-downloader.py

```diff
@@ -99,9 +99,9 @@ def format_filename(recording, file_type, file_extension, recording_type, record
     uuid = recording['uuid']
     topic = recording['topic'].replace('/', '&')
     rec_type = recording_type.replace("_", " ").title()
-    meeting_time = parse(recording['start_time'])
-    return '{} - {} UTC - {} - {}.{}'.format(
-        meeting_time.strftime('%Y.%m.%d'), meeting_time.strftime('%I.%M %p'), topic+" - "+rec_type, recording_id, file_extension.lower())
+    meeting_time = parse(recording['start_time']).strftime('%Y.%m.%d - %I.%M %p UTC')
```

Comment:
I also tidied up the `strftime` code to be a bit more DRY - set the date/time format once and be done